intermediate_source/dist_tuto.rst

```diff
@@ -631,7 +568,7 @@ with rank 0 and follow the procedure described above.
 
 .. code:: python
 
-    dist.init_process_group(init_method='tcp://10.1.1.20:23456', rank=args.rank, world_size=4)
+    dist.init_process_group(init_method='gloo://10.1.1.20:23456', rank=args.rank, world_size=4)
```

Comment:
I am not sure if the `init_method` for `tcp` and `gloo` accept the same format. @pietern could you please help to take a look?

Reply:
Thanks Shen. @pietern, context here is that this tutorial didn't work for me when I ran it with either the TCP or MPI backend.

We can make the edits to this tutorial in a separate commit.

Reply:
This is the initialization method. There is no `gloo://` initialization method, so I would assume running this will yield an error. The `backend` kwarg may be specified and can be set to `gloo`, `nccl`, or `mpi`. The initialization method is unrelated to which of the backends you pick.